Documentation/ldap-connector.md

```diff
@@ -0,0 +1,86 @@
+# Overview of the `ldap` Connector
+
+The `ldap` connector allows email/password based authentication hosted by dex, backed by a LDAP directory. All the configuration parameters required by the `ldap` connector are held in [Config struct](../connector/ldap/ldap.go).
+
+The connector executes two primary queries:
+
+1. Finding the user based on the end user's credentials.
+2. Searching for groups using the user entry.
+
+User entries are expected to have an email attribute (configurable through "emailAttr"), and a display name attribute (configurable through "nameAttr").
+
+### Setting the Configuration
```

Comment:
This can just be combined with the next section. Also we can probably just use "Configuration" as the header. Also two "#", not three.